File: src/BTech/BTech.Web/ClientApp/app/models/Serie.ts
import { Exercicio } from './Exercicio';
import { Conclusao } from './Conclusao';

export class Serie {
	public id: number;
	public tipoSerie: string;
	public exercicios: Exercicio[];
	public conclusoes: Conclusao[];
	public ativa: boolean;
}

File: src/BTech/BTech.DataAccess/Entities/Serie.cs
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using System;
using System.Collections.Generic;
using System.Text;

namespace BTech.DataAccess.Entities
{
	public enum TipoSerie
	{
		A,
		B,
		C
	}

	public class Serie
    {
		public int Id { get; set; }
		public List<Exercicio> Exercicios { get; set; }
		public List<Conclusao> Conclusoes { get; set; }
		public bool Ativa { get; set; }

		[JsonConverter(typeof(StringEnumConverter))]
		public TipoSerie TipoSerie { get; set; }
	}
}
